### ISSUE TITLE
App crashing when getting to fees setting to SEND

### ISSUE THREAD

**Issue by bitcoinheiro** [nunchuk-io/nunchuk-android#24]
This is a degoogled phone running with Tor proxy.

If I go to the "Local currency" tab on the app, it doesn't show any option to select from (maybe related).
Does the app requires clearnet and/or Google Play Services to work properly?

Steps to reproduce:

0. Open wallet
1. Press "Send"
2. Set value to send and press "Continue"
3. Set address to send and tag
4. Crashes if I press "Create transaction" or "Customize transaction"

From the crash log, I understand the problem is related to fee estimation.
Here's the crash log:
![image](https://github.com/nunchuk-io/nunchuk-android/assets/50273260/67e4a174-76bc-4965-b374-33de863d5875)

**Comment by JayceLuong** [comment]
Are you sure your device connect internet?

**Comment by bitcoinheiro** [comment]
Yes, the wallet is online. It can check the status of the UTXOs I have. Everything works fine except when sending. (kinda important feature 😁 )

**Comment by JayceLuong** [comment]
Yes, we support offline, but it needs to fetch the fee from the server at least once.

**Comment by bitcoinheiro** [comment]
But the wallet is online, just using Tor, so probably the fee server is identifying it as spam? But it should just say: "Cannot reach fee server, please connect to the internet or you're on your own to estimate the best fees" - something along those lines. Should not be an app crashing event

**Comment by JayceLuong** [comment]
Ok I will update the default value in this case
